examples.md

```diff
@@ -237,8 +237,61 @@ And finally there is the MeasurementOrFact extension table, which has attributes
+| in2017_v02_01012 | 2017-03-17 02:38:00 | Australasian Seabird Group, BirdLife Australia | in2017_v02_wov |
+
+| basisOfRecord    | recordedBy  | organismQuantity | organismQuantityType | occurrenceStatus |
+|------------------|-------------|------------------|----------------------|------------------|
+| HumanObservation | EJW+CRC+TAH | 2                | individuals          | present          |
+| HumanObservation | EJW+CRC+TAH | 1                | individuals          | present          |
+| HumanObservation | EJW+CRC+TAH | 1                | individuals          | present          |
+
+| decimalLatitude | decimalLongitude | coordinateUncertaintyInMeters | coordinatePrecision | footprintWKT                |
+|-----------------|------------------|-------------------------------|---------------------|-----------------------------|
+| -43.40741       | 147.45576        | 200                           | 0.0018              | POINT (147.45576 -43.40741) |
+| -45.98644       | 142.1445         | 200                           | 0.0018              | POINT (142.14450 -45.98644) |
+| -43.40728       | 147.45549        | 200                           | 0.0018              | POINT (147.45549 -43.40728) |
+
+| scientificNameID                          | scientificName                  | scientificNameAuthorship | vernacularName      |
+|-------------------------------------------|---------------------------------|--------------------------|---------------------|
+| urn:lsid:marinespecies.org:taxname:343991 | Morus serrator                  | (Gray,1843)              | Australasian Gannet |
+| urn:lsid:marinespecies.org:taxname:212648 | Pachyptila turtur               | (Kuhl,1820)              | Fairy Prion         |
+| urn:lsid:marinespecies.org:taxname:707545 | Chroicocephalus novaehollandiae | Stephens,1826            | Silver Gull         |
+
+**extendedMeasurementOrFact (eMoF) extension:**
+
+As shown in previous examples, the MeasurementOrFact extension table contains abiotic meaasurements or facts corresponding to an occurrence / sighting. Individual sightings and abiotic measurements are linked with `occurrenceID`. In the example dataset, the ENV-DATA consist of measurements taken during the moorings deployment at the SOTS site, at the time of the marine bird sightings. In addition to NERC vocabulary terms, authors also referenced the Australian Ocean Data Network (AODN) Discovery Parameter Vocabulary for *Depth* and *Sea Surface Temperature* as `measurementType`.
+
+| occurrenceID     | measurementID                     | measurementType                 | measurementTypeID                                           |
+|------------------|-----------------------------------|---------------------------------|-------------------------------------------------------------|
+| in2017_v02_00998 | in2017_v02_00998-depth            | Depth (m)                       | http://vocab.aodn.org.au/def/discovery_parameter/entity/574 |
```

Comment:
Any chance you can add the NERC equivalents for the AODN terms? These can be additional MoF records.